### ISSUE TITLE
Discussion, implement different languages

### ISSUE THREAD

**Issue by joevegan** [joevegan/ticketdesk#15]

**Comment by joevegan** [comment]
I was thinking of making some db tables and adding a setting on the system menu for language, the system class could be expanded to have a language dictionary that would match the keywords used in throughout and populate them based on the system setting in the db. Thoughts?

**Comment by joevegan** [comment]
To have a language added please provide translations for the below and we will get the language added in.

**System keywords**
- Ticket
- Client
- Subject
- Details
- Create
- Close
- Category
- Sub Category
- Assigned
- Status
- Reset
- New Ticket
- Recent Ticket
- Search
- Submit
- Open
- Closed
- Ticket Desk
- Tickets
- Reports
- System
- My Tickets
- All
- Waiting On Client
- Waiting On Agent
- Waiting On Other
- Report Name
- Start Date
- End Date
- Created by
- Categories
- Users
- Groups
- Log Out
- Log In
- Forgot?
- Register
- Username
- Password
- Add
- Delete
- Save

**Comment by joevegan** [comment]
Look into how wordpress does it with different html files
